core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-actions/template-builder-actions.component.ts

```diff
@@ -60,6 +63,17 @@ export class TemplateBuilderActionsComponent implements OnInit, OnDestroy {
         });
     }
 
+    ngOnChanges(): void {
+        this.group?.setValue(
+            {
+                header: this.layoutProperties.header,
+                footer: this.layoutProperties.footer,
+                sidebar: this.layoutProperties.sidebar
+            },
+            { emitEvent: false }
+        );
+    }
```

Comment:
1. Check the diff between set and patch value
2. Should we do it with the whole form?

Reply:
I think `patchValue` is the best option for this case, we don't want to update all the form all the time and that method does that under the hood